dns.go

```diff
@@ -80,6 +92,23 @@ func (c *Client) ListDomains() ([]Domain, error) {
 	return ds, nil
 }
 
+// NewDomain registers a new Domain
+func (c *Client) NewDomain(name string) (*Domain, error) {
+	url := fmt.Sprintf("/v2/dns")
```

Comment:
Why use fmt.Sprintf if not interpolating values?

Reply:
It was a bad copy / paste